### PR TITLE
Added initial home page image with text

### PR DESCRIPTION
**Pull request by matt-cha**

### CODE REVIEW

File: public/JJ-1396.jpg
Comment (joseph-w-valdez) [inline public/JJ-1396.jpg:1]: should format the name of this file

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,113 +1,128 @@
...
+    //           src="/vercel.svg"
+    //           alt="Vercel Logo"
+    //           className="dark:invert"
+    //           width={100}
+    //           height={24}
+    //           priority
+    //         />
+    //       </a>
+    //     </div>
+    //   </div>
```
Comment (joseph-w-valdez) [inline src/app/page.tsx:28]: don't comment out unused code

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,113 +1,128 @@
...
+    //   <div className="relative z-[-1] flex place-items-center before:absolute before:h-[300px] before:w-full before:-translate-x-1/2 before:rounded-full before:bg-gradient-radial before:from-white before:to-transparent before:blur-2xl before:content-[''] after:absolute after:-z-20 after:h-[180px] after:w-full after:translate-x-1/3 after:bg-gradient-conic after:from-sky-200 after:via-blue-200 after:blur-2xl after:content-[''] before:dark:bg-gradient-to-br before:dark:from-transparent before:dark:to-blue-700 before:dark:opacity-10 after:dark:from-sky-900 after:dark:via-[#0141ff] after:dark:opacity-40 sm:before:w-[480px] sm:after:w-[240px] before:lg:h-[360px]">
+    //     <Image
+    //       className="relative dark:drop-shadow-[0_0_0.3rem_#ffffff70] dark:invert"
+    //       src="/next.svg"
+    //       alt="Next.js Logo"
+    //       width={180}
+    //       height={37}
+    //       priority
+    //     />
+    //   </div>
```
Comment (joseph-w-valdez) [inline src/app/page.tsx:39]: same as above

File: src/app/page.tsx [hunk 3]
```diff
@@ -1,113 +1,128 @@
...
+    //       <h2 className="mb-3 text-2xl font-semibold">
+    //         Docs{" "}
+    //         <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">
+    //           -&gt;
+    //         </span>
+    //       </h2>
+    //       <p className="m-0 max-w-[30ch] text-sm opacity-50">
+    //         Find in-depth information about Next.js features and API.
+    //       </p>
+    //     </a>
```
Comment (joseph-w-valdez) [inline src/app/page.tsx:57]: same as above

File: src/app/page.tsx [hunk 4]
```diff
@@ -1,113 +1,128 @@
...
+    //       <h2 className="mb-3 text-2xl font-semibold">
+    //         Learn{" "}
+    //         <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">
+    //           -&gt;
+    //         </span>
+    //       </h2>
+    //       <p className="m-0 max-w-[30ch] text-sm opacity-50">
+    //         Learn about Next.js in an interactive course with&nbsp;quizzes!
+    //       </p>
+    //     </a>
```
Comment (joseph-w-valdez) [inline src/app/page.tsx:74]: same as above

File: src/app/page.tsx [hunk 5]
```diff
@@ -1,113 +1,128 @@
...
+    //       <h2 className="mb-3 text-2xl font-semibold">
+    //         Templates{" "}
+    //         <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">
+    //           -&gt;
+    //         </span>
+    //       </h2>
+    //       <p className="m-0 max-w-[30ch] text-sm opacity-50">
+    //         Explore starter templates for Next.js.
+    //       </p>
+    //     </a>
```
Comment (joseph-w-valdez) [inline src/app/page.tsx:91]: same as above

File: src/app/page.tsx [hunk 6]
```diff
@@ -1,113 +1,128 @@
...
+    //         <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">
+    //           -&gt;
+    //         </span>
+    //       </h2>
+    //       <p className="m-0 max-w-[30ch] text-balance text-sm opacity-50">
+    //         Instantly deploy your Next.js site to a shareable URL with Vercel.
+    //       </p>
+    //     </a>
+    //   </div>
+    // </main>
```
Comment (joseph-w-valdez) [inline src/app/page.tsx:110]: same as above

File: src/app/page.tsx [hunk 7]
```diff
@@ -1,113 +1,128 @@
...
+    <div className="relative w-full h-screen">
+      <img
+        src="/JJ-1396.jpg"
```
Comment (joseph-w-valdez) [inline src/app/page.tsx:113]: import this image like `import banner from '/rel-path.jpg` and use `banner` as the `src` here

**Review by joseph-w-valdez — CHANGES_REQUESTED** [review]

File: public/main-image-1.jpg
Comment (joseph-w-valdez) [inline public/main-image-1.jpg:1]: should omit the `-1`. if you're gonna use this on the main page, `banner.jpg` and `main.jpg` are probably better. no need to include `image` in the file name as it's already a `.jpg` file

File: src/app/contact/page.tsx
Comment (joseph-w-valdez) [inline src/app/contact/page.tsx:1]: why does this file exist if this PR is just for the home page

**Review by joseph-w-valdez — CHANGES_REQUESTED** [review]

File: src/app/page.tsx [hunk 8]
```diff
@@ -1,113 +1,22 @@
...
-          className="relative dark:drop-shadow-[0_0_0.3rem_#ffffff70] dark:invert"
-          src="/next.svg"
-          alt="Next.js Logo"
-          width={180}
-          height={37}
-          priority
-        />
+    <div className="relative w-full h-screen">
+      <img
+        src={main.src}
```
[the remark this answers is not in the document]
Comment (matt-cha) [reply src/app/page.tsx:7]: It was giving me a TS Error so I looked into it and it's accessing the StaticImageData type vs what it actually wants which is a string. This prevents the TS error on my end!

File: src/app/contact/page.tsx [hunk 9]
```diff
@@ -0,0 +1,11 @@
+"use client";
```
[the remark this answers is not in the document]
Comment (matt-cha) [reply src/app/contact/page.tsx:1]: Removed the file, will put this into a separate branch

File: src/app/page.tsx [hunk 10]
```diff
@@ -1,113 +1,22 @@
...
-          </p>
-        </a>
+      <div className="flex items-center justify-center">
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Ex dicta
+          dolores suscipit fuga a illum esse facilis laborum ratione mollitia,
+        </p>
       </div>
-    </main>
+    </div>
```
[the remark this answers is not in the document]
Comment (matt-cha) [reply src/app/page.tsx:20]: Root div to main, got it. Is this more a semantic, organizational code practice?

File: public/main.jpg
Comment (joseph-w-valdez) [inline public/main.jpg:1]: is this meant to be 20MB? quite a large file for the splash page

File: src/app/page.tsx (hunk 10 above)
[the remark this answers is not in the document]
Comment (joseph-w-valdez) [reply src/app/page.tsx]: important for both reasons yeah. not necessarily root div, but the -main- content on the page. you can have other sibling divs here like aside elements, etc.

**Review by joseph-w-valdez — APPROVED** [review]

**Pull request merged**